### ISSUE TITLE
Picovoice account gets deleted

### ISSUE THREAD

**Issue by RetoHB** [DevMiser/DaVinci#19]
Picovoice keeps deleting my accounts. This already happened twice, every time about a day after I opened the account.
Anybody having the same issue?

"An account registered with 'xxxxx' has been removed by the Picovoice Audit System. Accounts can be removed in cases of Picovoice Terms of Use violations, such as suspicious activities, creating multiple accounts, or providing false information."

**Comment by DevMiser** [comment]
This has never happened to me.  

You may want to try to contact Picovoice directly.  I think their email is hello@picovoice.ai.

**Comment by RetoHB** [comment]
I tried that, but they are not responding on that email.

**Comment by EA914** [comment]
You know, for speech to text you can use OpenAI's whisper model. This would be better in my opinion so you only are hitting one API endpoint.

Check my github repository for a code example. I suggest using this for speech-to-text and transcribe your prompt, send to ChatGPT to get a response, then use OpenAI again for text-to-speech!

**Comment by DevMiser** [comment]
Yes, OpenAI's Whisper can be used for the speech to text.  However, you still need to have wake word capability for DaVinci, which Whisper does not provide.  DaVinci currently uses Picovoice Porcupine for detecting its name. Rhasspy may provide another wake word option if you cannot or do not want to use Picovoice.  I prefer the responsiveness of Picovoice for both wake word and speech-to-text on a Raspberry Pi.  And the PicoVoice options are currently free of charge for hobbyists as opposed to OpenAI's now charge per use plan. If you also want to use OpenAI for text-to-speech, you can use the DaVinciAltVoice.py version on this repository.

**Comment by CoffeCodingJava** [comment]
> Picovoice keeps deleting my accounts. This already happened twice, every time about a day after I opened the account. Anybody having the same issue?
> 
> "An account registered with 'xxxxx' has been removed by the Picovoice Audit System. Accounts can be removed in cases of Picovoice Terms of Use violations, such as suspicious activities, creating multiple accounts, or providing false information."

Hello, same issue here. Did they get back to you on that?
